src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit" />
+    <PackageReference Include="Basic.Reference.Assemblies.Net60" />
```

Comment:
why?

Reply:
Because analyzer tests can be run as code style tests and when I first tried to run newly added tests as code style ones, they failed due to missing reference
![devenv_XleaJHH53q](https://github.com/dotnet/roslyn/assets/70431552/720149dc-e450-414c-8fdc-b8feb078a4c9)